rollbar-agent

```diff
@@ -529,7 +529,7 @@ class ScannerThread(threading.Thread):
             log.debug("File %s new pos %d", filename, new_pos)
             file_state['pos'] = new_pos
 
-        if self.config['delete_processed_files'] and new_pos == os.stat(filename).st_size:
+        if app['config']['delete_processed_files'] and new_pos == os.stat(filename).st_size:
```

Comment:
Oooh, ya. Otherwise you could have put it into the `DEFAULT` section but I like this solution better.